web/client/translations/data.en-US

```diff
@@ -27,6 +27,7 @@
         "showMore": "Show more...",
         "collapse": "Collapse",
         "expand": "Expand",
+        "geostory": "GeoStory",
```

Comment:
This is overridden by the full object, the other translations ported are not failing because they found out this...

I have a translation ready, @allyoucanmap I'm going to  send them, but if you can, you should revert he translations for the other languages